src/exampleTypes/ResultTable/Header.js

```diff
@@ -192,8 +206,13 @@ let Header = ({
           <NestedPicker
             options={addOptions}
             onChange={field => {
-              if (!_.contains(field, includes))
-                mutate({ include: [...includes, field] })
+              addColumn(
+                mutate,
+                i => i + 1, // Add to the right of the clicked column
```

Comment:
since we're only adding to the right, there's probably no value in the computeSideIndex abstraction yet